### PR TITLE
Get and Edit Mentor Availability 

### LINKED ISSUE #86: Write endpoints for getting and editing mentor availability
#### Description
What the title says

#### Tasks
- [ ] GET availability between a start and end time given a mentor ID
- [ ] PUT availability for a certain day (start and end time?? idk u choose how u wanna implement that)

### PR DESCRIPTION
**Pull request by AzureQDragon**
## Status:
:rocket: Ready

## Description :star2:
Endpoints for Availability

Fixes #86 

Get Request for availability
Screenshot 2020-12-03 180800

Put Request for availability
Before
Screenshot 2020-12-03 181000
Request
Screenshot 2020-12-03 181737
After
image

Get set days (probably need to change name of path would like suggestions for what it should be):
image

## TODOs :star:
- [x] GET endpoint
- [x] PUT endpoint
- [x] Tests
- [x] Documentation

### CODE REVIEW

File: backend/api/views/availability.py [hunk 1]
```diff
@@ -0,0 +1,63 @@
+from flask import Blueprint, request, jsonify
+from api.models import Availability, MentorProfile
+from api.core import create_response, serialize_list, logger
+
+availability = Blueprint("availability", __name__)
+
+
+@availability.route("/availability/<id>", methods=["GET"])
+def get_availability(id):
+
```
Comment (chyku) [inline backend/api/views/availability.py:10]: total nitpick but why is there this whitespace here

File: backend/api/views/availability.py [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+            end_time=availability.get("end_time").get("$date"),
+        )
+        for availability in data
+    ]
+    mentor.save()
+    return create_response(status=200, message=f"Success")
+
+
+@availability.route("/availability/setdays/<id>", methods=["GET"])
+def get_set_availability_days(id):
```
Comment (chyku) [inline backend/api/views/availability.py:43]: can you briefly comment what this does

**Review by chyku — APPROVED** [review]
lgtm, pls comment tho

**Review by AngelaLuo49021 — APPROVED** [review]
looks good!

**Pull request merged**